types/interfaces.ts

```diff
@@ -182,3 +182,6 @@ export enum UploadModes {
   NewModel = 'newModel',
   NewVersion = 'newVersion',
 }
+
+export enum DateStringBrand {}
```

Comment:
We shouldn't export this enum in order to prevent counterfeit `DateString`s from being created elsewhere. The point of the "brand" is to make `DateString` unique.